### DISCUSSION TITLE
2023-07-18

### DISCUSSION THREAD

**Discussion by woo-jk** [woowacourse-teams/2023-haru-study#93] [daily_meeting]
### 내일의 할 일
- 백엔드
  - 오전 중으로 스터디 조회 기능 PR merge
  - 스터디 진행 기능 완성
  - 스터디 참여 기능 완성

### 언젠가 해야할 일
- 기획 변경된 이유 정리
- DNS 어떻게 할지
- UX 저자 워크숍 사전 미션(~7/25 화)

**Comment by aak2075** [comment]
### 체크인

9점 - 조금 피곤하지만 화이팅해봅시다

### 체크아웃

10점 - 잘 진행되고 있는 것 같아 기쁘네요

**Comment by MoonJeWoong** [comment]
### 체크인
9점 - 어제 마주쳤던 테스트 문제를 해결해서 기분은 좋은데 찝찝한 부분이 남아있어서 1점 감점했습니다!

### 체크아웃
9점 - 오늘 생각보다 개발이 잘 풀려서 좋았습니다! 그런데 조금 피곤해서 1점 그대로 감점했습니다.

**Comment by woosung1223** [comment]
### 체크인
7점 - 디버깅 쉽지 않네요.. 아직도 머리가 뜨끈합니다

### 체크아웃
8점 - 힘들긴 하지만 해야할 일은 다 한 거 같아서 기분이 좋읍니다

**Comment by nlom0218** [comment]
### 체크인

9점 - 할 일이 조~~끔 쌓여있어서 -1 

### 체크아웃

10점 - 합성 컴포넌트,,!! 재미지다

**Comment by woo-jk** [comment]
### 체크인
9점 - 컨디션 굳 하지만 비와서 -1

### 체크아웃
7점 - 컴포넌트 짜는데 고민이 너무많타

**Comment by jaehee329** [comment]
### 체크인
9점 - 월요일보다는 조금 피곤해지고 있어

### 체크아웃
8점 - 작업은 잘 됐지만 어제보다는 피곤한 느낌

**Comment by yeopto** [comment]
### 체크인

10점 - 얼른 Dropdown 컴포넌트 만들 생각에 행복합니다.

### 체크아웃

5점 - compound component 너무 어렵다. Dropdown....
